_projects/enhance-and-improve-coala-community-and-projects-website.md

```diff
@@ -0,0 +1,160 @@
+tags:
+  - community
+  - projects
+  - webservices
+  - AngularJS
+  - Django
+  - Python
+---
+### Overview
+The motivation here, is to enhance and improve the websites ([community](https://community.coala.io) and
+[projects](https://projects.coala.io)) for the coala community, for newcomers
+who wish to start contributing to coala open-source organization and for other
+organizations. The hurdle that newcomers mainly come across is the invitation
+to the organization before making any contributions, so this project adds a
+form for newcomers who wish to join the coala community. Also, to give a
+motivation to existing contributors, coala has created a `Promotion form` which
+will promote them to `coala Developer` status. To reduce the workload for
+maintainers, this project includes the development of form with some
+rudimentary checks that will promote the coala-newcomer to coala-developer.
+Moreover, this project adds major enhancements to websites that will help
+newcomers as well as developers to know more about this community and about
+what they have contributed to it.
+
+### Goals
+
+* Add calendar app to show timeline for various events
+* Add a one-stop tab for all google forms
+* Add contributors geolocation
+* Add *Login/SignUp* using GitHub
+* Add organization teams
+* Advanced filters
+* Automate welcoming process for newcomer
+* Automate newcomer promotion
+* Create new database models to manage data
+* Ease navigatibility in websites
+* Improve the **UI/UX of websites** to make more interactive
+* Motivating newcomers & developers with -
+  * modifications in Gamification Leaderboard
+  * adding a tab of **GSoC** in projects website
+* New coala-community logo
+* Contributors profile page
+* Admin/Maintainers dashboard
+
+### Milestones
+
+#### PREPARATION/BONDING
+
+* Detailed research on what **new cron jobs** or **API endpoints** need to be
+  added in [coala webservices](https://gitlab.com/coala/landing/).
+* Discussion on improvements to be made in coala [community website](http://community.coala.io/) and
+  [projects website](http://projects.coala.io/)
+* Research on new models to be created (if any) and adding new fields to
+  existing db models in coala community website **db models**.
+* *Created* and *Merged* a cEP of how the new enhancements will be implemented
+  and how will it be beneficial to [coala community](https://github.com/orgs/coala/teams/coala-newcomers).
+
+#### CODING PHASE 1
+
+1. Implementation of methods in **coala [webservices](https://gitlab.com/coala/landing)**
+2. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create new **[Community Logo](https://github.com/coala/community/issues/55) + Header Image of README.md**
+
+    b. Add information about new endpoints at coala-webserivces homepage.
+
+    c. Add [contributor's Geolocation](https://github.com/coala/community/issues/18)
```

Comment:
I had a pr for this https://github.com/coala/community/pull/81, why it didn't get merged and how do you plan to do this now?

Reply:
Sorry, I haven't seen that PR 😢 
but What I was planning to achieve is simply creating a `cron Job` for updating the `cluster map` that is being created on the basis of the location of our community members (if added in their profile). It will also require an OAuth token(IMO) because of querying thousands of users. 
Then Once there is a cluster map on the `coala-webservices` we can show that on our community website.